pkg/machinery/nethelpers/match_operator.go

```diff
@@ -0,0 +1,18 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package nethelpers
+
+//go:generate enumer -type=MatchOperator -linecomment -text
+
+// MatchOperator is a netfilter match operator.
+type MatchOperator int
+
+// MatchOperator constants.
+//
+//structprotogen:gen_enum
+const (
+	OperatorEqual    MatchOperator = 0 // ==
```

Comment:
maybe add a comment that this matches the nftables library

Reply:
I'd love to, but this will break enumer